README.md

```diff
@@ -5,7 +5,11 @@
 
 <h1 align="center">MUI X</h1>
 
-MUI X is a collection of advanced UI components, including a [data grid](https://mui.com/x/react-data-grid/) and [date and time pickers](https://mui.com/x/react-date-pickers/getting-started/).
+MUI X is a collection of advanced UI components, including:
+
+- [Data Grid](https://mui.com/x/react-data-grid/)
```

Comment:
Maybe makes sense adding imgs for the components

Reply:
Make sense, but the effort of introducing a component in the markdown page to have nice-looking images might does not worth the benefit.

For me it's already an improvement as it is 👍

Reply:
I realized the list was duplicated, so I removed it altogether.